plugins/upload-assets-to-s3/test/tasks/upload-assets-to-s3.test.ts

```diff
@@ -105,4 +105,22 @@ describe('upload-assets-to-s3', () => {
       expect(e.details).toEqual(mockError)
     }
   })
+
+  // HACK:20231006:IM make sure hack to support Doppler migration works
+  it('should fallback to uppercase environment variable', async () => {
+    const task = new UploadAssetsToS3(logger, {
+      ...defaults,
+      directory: testDirectory
+    })
+    // must use delete to ensure an environment variable is undefined, setting
+    // 'undefined' will just stringify the word
+    delete process.env.aws_access_hashed_assets
+    process.env.AWS_ACCESS_HASHED_ASSETS = '1234'
```

Comment:
**praise:** This is a clear test which helps explains what is happening! Thanks for adding this 🙌